tutorials/websocket-chat-app-js.mdx

```diff
@@ -539,7 +537,7 @@ const websocket = new WebSocket(wsUri);
 
 Now that we've changed the connection URL, our compiled assets in our Rust folder will work on the local Rust run without us having to use our local Vite deployment! However, if we try to run our front end by itself, it won't connect as this connection string relies on using the local Rust project - but you can change the connection string as required.
 
-Now we can update our main function and router together so that the router will use a `SpaRouter` function provided by `axum_extra`, as well as use the static file path:
+Now we can update our main function and router together so that the router will use tower_http's ServeDir function for serving static files:
```

Comment:
```suggestion
Now we can update our main function and router together so that the router will use `tower_http`'s `ServeDir` function for serving static files:
```